File: src/Other/Internals/ChildWindowHelper.cs
using System.Windows;
using System.Windows.Controls;

namespace OpenSilver.Samples.Showcase
{
    static class ChildWindowHelper
    {
        public static void ShowChildWindow(UIElement content)
        {
            var childWindow = new ChildWindow()
            {
                Content = content,
                Style = (Style)App.Current.Resources["ViewMoreChildWindow_Style"]
            };
            childWindow.Show();
        }
    }
}

File: src/Other/Internals/ChildWindowHelper.fs
namespace OpenSilver.Samples.Showcase

open System.Windows
open System.Windows.Controls

type ChildWindowHelper() =
    static member ShowChildWindow (content: UIElement) =
        let childWindow = new ChildWindow()
        childWindow.Content <- content
        childWindow.Style <- App.Current.Resources["ViewMoreChildWindow_Style"] :?> Style
        childWindow.Show()

